### PR TITLE
Quiz question screen v2

### PR DESCRIPTION
**Pull request by dvira**
- Updated `CourseEditorScreen` css vars.
- Removed `interface` file and move its content into `utils` file.
- Updated `QuestionScreenForm` to match v2 design.

### CODE REVIEW

File: src/components/Screen-v2/CourseEditorScreen/CourseOutline/CourseOutlineTab/CourseOutlineDetailsPanel/QuizEditForm/Answers.tsx [hunk 1]
```diff
@@ -39,72 +47,57 @@ function Answers({
...
+      <div className={styles.addAnswerWrapper}>
+        <WMButton
+          className={styles.addBtn}
+          variant={WMButtonVariant.Text}
+          iconComponent={<WMIconPlus size={10} />}
+          onClick={onAddAnswer}
+        >
+          Add Answer
+        </WMButton>
+        {hasError && <span className={styles.errorMessage}>{ANSWERS_VALIDATION_ERROR}</span>}
```
Comment (nunibaranes) [inline src/components/Screen-v2/CourseEditorScreen/CourseOutline/CourseOutlineTab/CourseOutlineDetailsPanel/QuizEditForm/Answers.tsx:97]: Please use `WMMessage`

File: src/components/Screen-v2/CourseEditorScreen/CourseOutline/CourseOutlineTab/CourseOutlineDetailsPanel/QuizEditForm/Answers.tsx [hunk 2]
```diff
@@ -39,72 +47,57 @@ function Answers({
...
-        ) : (
-          <WMCheckbox
-            className={classes['multiple-select-field']}
-            value={answer.id}
-            checked={answer.isCorrect}
+          <WMTextField
+            maxLength={200}
+            placeholder="Text"
+            value={answer.text}
+            error={fieldErrorMessage(answer.text)}
```
Comment (nunibaranes) [inline src/components/Screen-v2/CourseEditorScreen/CourseOutline/CourseOutlineTab/CourseOutlineDetailsPanel/QuizEditForm/Answers.tsx:72]: Screen Shot 2021-01-10 at 14 27 50

When hasError, it looks bad.

File: src/components/Screen-v2/CourseEditorScreen/CourseOutline/CourseOutlineTab/CourseOutlineDetailsPanel/QuizEditForm/style.module.scss [hunk 3]
```diff
@@ -38,127 +39,96 @@
...
         }
-      }
 
-      /* Radio styles **/
-      :global(.ant-radio-checked) :global(.ant-radio-inner) {
-        background-color: var(--green);
+        .icon {
+          background-color: var(--wmWhite);
+          border: 1px solid var(--wmGrayLight);
+          border-radius: 50%;
```
Comment (nunibaranes) [inline src/components/Screen-v2/CourseEditorScreen/CourseOutline/CourseOutlineTab/CourseOutlineDetailsPanel/QuizEditForm/style.module.scss:88]: I'd consider changing the shape to a square according to the answer selection's type.

**Review by nunibaranes — CHANGES_REQUESTED** [review]
Looks great!
I found one bug you should fix.
Also, I Left a few suggestions.

File: src/components/Screen-v2/CourseEditorScreen/CourseOutline/CourseOutlineTab/CourseOutlineDetailsPanel/QuizEditForm/style.module.scss (hunk 3 above)
Comment (dvira) [reply src/components/Screen-v2/CourseEditorScreen/CourseOutline/CourseOutlineTab/CourseOutlineDetailsPanel/QuizEditForm/style.module.scss:89]: Still awaiting Adi's instructions regarding the icon in multiple selection mode.
For now leaving this as is, and once I have her input, will fix if necessary.

**Review by dvira — COMMENTED** [review]
@nunibaranes Thanks for your input!

File: src/components/Screen-v2/CourseEditorScreen/CourseOutline/CourseOutlineTab/CourseOutlineDetailsPanel/QuizEditForm/Answers.tsx (hunk 1 above)
Comment (dvira) [reply src/components/Screen-v2/CourseEditorScreen/CourseOutline/CourseOutlineTab/CourseOutlineDetailsPanel/QuizEditForm/Answers.tsx]: Done

**Review by nunibaranes — APPROVED** [review]
Approved!!!

**Pull request merged**